src/components/BlockSection.svelte

```diff
@@ -2,7 +2,7 @@
   export let title: string;
 </script>
 
-<div class="content-block flex flex-col gap-4">
+<div class="content-block column">
```

Comment:
This seems to break the styling for other components like Provider Login.  I added additional styles and updated the class on ProviderLogin but it looks slightly different than the Figma.